.github/workflows/build-release-devnet.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and Release Devnet
+
+on:
+  push:
+    branches: [ "devnet" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
```

Comment:
Actually we do not have any git submodules yet.

Reply:
Ok, it was only an update to the actions version, no more

Reply:
I mean these extra lines moved from repo to repo, but have not so much sense.

Reply:
Fixed